pkgs/by-name/ge/gema/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenv,
+  fetchgit,
+  installShellFiles,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gema";
+  version = "2.0";
+
+  src = fetchgit {
+    url = "https://git.code.sf.net/p/gema/code.git";
+    rev = "v2-0";
```

The necessary `rev` can be derived from `version` as follows:
```suggestion
    rev = "v${lib.replaceStrings [ "." ] [ "-" ] finalAttrs.version}";
```

---

Switched to fetching the tarball rather than the git tag as suggested below, so finalAttrs.version can be used directly.